invenio_vocabularies/contrib/subjects/config.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 Northwestern University.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Subjects configuration."""
+
+from flask_babelex import lazy_gettext as _
+from invenio_records_resources.services import SearchOptions
+from invenio_records_resources.services.records.components import DataComponent
+
+from ...services.components import PIDComponent
+from ...services.querystr import FilteredSuggestQueryParser
+
+
+class SubjectsSearchOptions(SearchOptions):
+    """Search options."""
+
+    suggest_parser_cls = FilteredSuggestQueryParser.factory(
+        filter_field='scheme',
+        fields=[  # suggest fields
+            'subject^100',
+            'subject._2gram',
```

Comment:
This works for `search_as_you_type` fields. I've added that to v7 subject (not available in v6)